features/_scicommon/exec.config

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -Eeuox pipefail
+
+# needs to be in line with ../khost/exec.late K8S_VERSION* variables
+K8S_VERSION=v1.30.7
```

Comment:
Disclaimer: I am not initimately familiar with the build process. Considering the `/builder/features/` reference below, would something like this be possible to avoid hardcoding?

```bash
K8S_VERSION=$(grep '^K8S_VERSION=' "/builder/features/khost/exec.late" | cut -d'=' -f2)
```

Reply:
Ah... not right now since `capi` is not a hard dependency for the `_scicommon` feature!

Reply:
had to resolve this differently